### PR TITLE
Some event handler unit tests

### PR DESCRIPTION
**Pull request by adrianosela**

### CODE REVIEW

File: api/handlers/event.js [hunk 1]
```diff
@@ -16,6 +17,10 @@ let getEvent = async function(req, res) {
             return res.status(400).send("No event id in query string");
         }
 
+        if (!mongodb.ObjectID.isValid(eventId)) {
+            return resp.status(400).send("id provided is invalid");
+        }
+
```
Comment (israeltq) [inline api/handlers/event.js]: Why do we need this?

Comment (adrianosela) [reply api/handlers/event.js]: because otherwise we throw a 500 after mongo throws an exception

File: api/handlers/event.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 const logger = require("tracer").console();
 const validator = require("../validator/validator");
 const eventHelpers = require("../utils/event");
+var mongodb = require("mongodb");
```
Comment (israeltq) [inline api/handlers/event.js]: I dont like the changes in this file, if we do this, then we need to change it everywhere else. I would advise against that

File: api/handlers/event.js (hunk 1 above)
Comment (israeltq) [reply api/handlers/event.js]: If we are going to add this, then pls add it to validator and check all calls (i.e. dont add it to the handler files but just create a validator function)

File: api/handlers/event.js (hunk 2 above)
Comment (israeltq) [reply api/handlers/event.js]: same here, give this to validator

File: api/handlers/event.js [hunk 3]
```diff
@@ -145,8 +151,7 @@ let updateEvent = async function(req, res) {
             event[property] = newEvent[property];
         }
         await event.save();
-
```
Comment (israeltq) [inline api/handlers/event.js]: leave this space

File: api/handlers/event.js [hunk 4]
```diff
@@ -136,6 +141,7 @@ let updateEvent = async function(req, res) {
         }
 
         let event = await Event.findById(newEvent._id);
+
```
Comment (israeltq) [inline api/handlers/event.js]: remove this space

**Pull request merged**